### PR TITLE
Slow down

### PR DESCRIPTION
**Pull request by avlund**
Very zen

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
         ctx     = canvas.getContext('2d'),
         ucanvas = get('upcoming'),
         uctx    = ucanvas.getContext('2d'),
-        speed   = { start: 0.6, decrement: 0.005, min: 0.1 }, // how long before piece drops by 1 row (seconds)
+        speed   = { start: 1.0, decrement: 0.005, min: 0.7 }, // how long before piece drops by 1 row (seconds)
```
Comment (mortenskou) [inline index.html]: WHY IS THIS SO SLOW?

Comment (mortenskou) [inline index.html]:
```suggestion
        speed   = { start: 0.0001, decrement: 0.005, min: 0.1 }, // how long before piece drops by 1 row (seconds)
```

File: index.html [hunk 2]
```diff
@@ -75,7 +75,7 @@
         ctx     = canvas.getContext('2d'),
         ucanvas = get('upcoming'),
         uctx    = ucanvas.getContext('2d'),
```
Comment (mortenskou) [inline index.html]:
```suggestion
        uctx    = ucanvas.getContext('2d'),
        alert("Pwned");
```

File: index.html (hunk 1 above)
Comment (avlund) [reply index.html]: NOOO

**Pull request merged**